addons/point_of_sale/static/src/app/screens/product_screen/product_screen.js

```diff
@@ -371,9 +371,16 @@ export class ProductScreen extends Component {
             .filter((product) => !this.getProductListToNotDisplay().includes(product.id))
             .slice(0, 100);
 
-        return list.sort(function (a, b) {
-            return a.display_name.localeCompare(b.display_name);
-        });
+        return this.searchWord !== "" 
+            ? list
+            : list.sort((a, b) => a.display_name.localeCompare(b.display_name));
+    }
+
+    product_search_string(product) {
+        const fields = ['display_name', 'description_sale', 'barcode'];
+        return fields.map(field => product[field] || "")
```

Comment:
```suggestion
        return fields.map(field => unaccent(product[field] || "", false))
```
Remove the accents and make the search case-insensitive.